### DISCUSSION TITLE
[ISSUE] RubyMine not showing Intellisense for Rswag methods

### DISCUSSION THREAD

**Discussion by devsheva** [rswag/rswag#762] [General]
## Describe the bug
RubyMine IDE doesn't show code completion helpers for Rswag methods.

## Steps to Test or Reproduce
Just install the rspec and rswag gems and try to write a test.

## Expected behavior
RSwag methods should be available to code completion.

## Additional context
Probably i'm missing something in the configuration of IDE or rspec. I don't know, it's more a question than an issue/bug i guess.

## Dependency versions
The version of are you using for:
* Rswag: 2.13.0
* RSpec: 3.12.0
* Rails: 7.1.3
* Ruby: 3.1.3

## Relates to which version of OAS (OpenAPI Specification)
- [x] OAS3.1

**Comment by araccaine** [comment]
Unfortunately we are out of luck: 

> Unfortunately, we have no near future plans to support rswag in RubyMine.

https://youtrack.jetbrains.com/issue/RUBY-24008/add-support-for-rswag-in-RubyMine#focus=Comments-27-9022187.0-0
